packages/esm-care-panel-app/src/care-panel/care-panel.component.tsx

```diff
@@ -65,7 +65,7 @@ const CarePanel: React.FC<CarePanelProps> = ({ patientUuid, formEntrySub, launch
           </div>
         </CardHeader>
         <div style={{ width: '100%', minHeight: '20rem' }}>
-          {switchItem?.name === 'HIV Program' ? (
+          {programEnrolled === 'HIV Program' ? (
             <>
```

Comment:
we dont need this extra fragement

Reply:
@jabahum This has been adjust in the preceding comment

Reply:
What about TB